requirements.txt

```diff
@@ -9,3 +9,4 @@ pyOpenSSL
 flask
 lockfile
 bs4
+pycurl
```

Comment:
did you use `pycurl` in anywhere?

Reply:
Hey, 
I used `pycurl` in ntlm authentication brute.

Reply:
Thanks, I thin i's just ui enhancements. I'll review it and get back to you soon, thanks for the nice ui.

Reply:
Sorry for the confusion created. I thought to include this in the same PR instead of creating a seperate PR. I can change the title of the PR to a more informative one.